portfolio.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+<body>
+    <header class="nav-container">
+        <nav>
+            <div class="header-bar">
+               <div class="haftamu">Haftamu</div>
+               <div class="allign"><img src="bar/Union.png" alt="">
+            </div>
+            </div>
+        </nav>
+    </header>
+    <div class="top-part">
+    <section class="section1">
+        
+            <div class="Developer">
+                <div class="software-developer">
+                    <h1 class="hey">Hey There.
+                        I’m Haftamu</h1>
+                    <h2 class="soft">I’am a Software Developer</h2>
+                    <p class="info">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+                
+        
+                    <div class="social-media">
+                        <ul>
+                            <li> 
+                                <a href="#"><img class="social" src="socialmedia/github.png" alt=""></a>
+                          </li>
+                            <li>
+                                <a href="#"><img class="social" src="socialmedia/linkidin.png" alt=""></a>
+                              </li>
+                                
+                            <li><a href="#"><img class="social" src="socialmedia/tump.png" alt=""></a>
+                            </li>
+                                
+                            <li><a href="#"><img class="social" src="socialmedia/tiwtter.png" alt=""></a>
+                                </li>
+                            <li><a href="#"><img class="social" src="socialmedia/m.png" alt=""></a>
+                                </li>
```

Comment:
- [ ] The design of these icons is a bit off from that of the Figma. One of them is the 'M' icon is blue and yet it is supposed to be white. Please change all the icons to look like those in the Figma 👍 
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/165341500-bbf7d9d4-055b-415e-8e67-fb2594601fa4.png)

### Figma
![image](https://user-images.githubusercontent.com/45460192/165341566-3f09ef4f-b034-4a04-92d3-00518852095a.png)